ex_0811/Batch.java

```diff
@@ -0,0 +1,57 @@
+package ex_0811;
+
+public class Batch {
+    private String batchName;
+    private String batchDate;
+    private Interviewer interviewer;
+    private Candidate[] candidates;
+
+    public Batch(String batchName, String batchDate) {
+        this.batchName = batchName;
+        this.batchDate = batchDate;
+    }
+
+    public String getBatchName() {
+        return batchName;
+    }
+
+    public boolean setBatchName(String batchName) {
+        if (batchName != null) {
+            this.batchName = batchName;
+            return true;
+        } else {
+            System.out.println("Please enter name again: ");
```

Comment:
No need to println here, should check condition in class main and print in class main.